docker-compose-generator/docker-fragments/phoenixd.yml

```
@@ -0,0 +1,22 @@
+version: "3"
+
+services:
+  phoenixd:
+    image: acinq/phoenixd:0.5.1
+    container_name: phoenixd
+    restart: unless-stopped
+    networks:
+      - default
+    command: ["--chain=${NBITCOIN_NETWORK:-regtest}"]
+    expose:
+      - "9740"
+    depends_on:
+      - btcpayserver
+    volumes:
+      - "phoenixd_datadir:/phoenix"
+
```

I think we may need this section to define the default connection string, with a hardcoded api password.

```suggestion
  btcpayserver:
    environment:
      BTCPAY_BTCLIGHTNING: "type=phoenixd;server=http://phoenixd:8080;password=xxx"
    volumes:
      - "phoenixd_datadir:/etc/phoenix"
    links:
      - phoenixd
```

---

Done.